testing/test_warnings.py

```diff
@@ -543,7 +543,7 @@ def assert_result_warns(result, msg):
 
     def test_tuple_warning(self, testdir):
         testdir.makepyfile(
-            """
+            """\
```

Comment:
Please do not change code unnecessary, especially when it is only to please your editor in some way (IIRC).
For all others this adds visual disbalance, and often gets highlighted in a special way (line continuation).

Reply:
?